CHANGELOG.md

```diff
@@ -33,7 +33,7 @@
 
 ### Improvements
 
-- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+- Fixing logging in KEDA operator and properly set `ScaledObject.Status` in case there is a problem in a ScaledObject's trigger ([#2603](https://github.com/kedacore/keda/issues/2603))
```

Comment:
```suggestion
- **General** - Fixing logging in KEDA operator and properly set `ScaledObject.Status` in case there is a problem in a ScaledObject's trigger ([#2603](https://github.com/kedacore/keda/issues/2603))
```

Reply:
use `**General**: ` instead of `**General** -` please

Reply:
You are right, thank you!